_layouts/minerva-1-5.html

```diff
@@ -25,11 +24,17 @@
     <script src="https://cdn.jsdelivr.net/npm/es6-promise@4/dist/es6-promise.auto.min.js"></script> 
     <script src="https://cdn.jsdelivr.net/npm/minerva-browser@3.15.4/build/bundle.js"></script>
     <script>
+        const exhibit = {{ exhibit | jsonify }};
+        const image_path = "{{ page.images }}";
+        /* Overwrite image path from config */
+        if (image_path.length > 0) {
+          exhibit.Images.forEach(img => img.Path = image_path);
+        }
```

Comment:
This is the section in the `minerva-1-5` template that overrides the exhibit.json path if the "images" key is present.